### PR TITLE
Update chatops

### PR DESCRIPTION
**Pull request by osterman**
## what
* change default permission to `triage`

## why
* security

### CODE REVIEW

File: .github/workflows/chatops.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  issue_comment:
+    types: [created]
+
+jobs:
+  default:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Handle common commands"
+        uses: cloudposse/actions/github/slash-command-dispatch@0.15.0
```
Comment (Nuru) [inline .github/workflows/chatops.yml]: Do you want to use the new slash command?
```suggestion
        uses: cloudposse/actions/github/slash-command-dispatch@0.16.0
```

File: .github/workflows/chatops.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          permission: triage
+          issue-type: pull-request
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout commit"
+        uses: actions/checkout@v2
+      - name: "Run tests"
+        uses: cloudposse/actions/github/slash-command-dispatch@0.15.0
```
Comment (Nuru) [inline .github/workflows/chatops.yml]:
```suggestion
        uses: cloudposse/actions/github/slash-command-dispatch@0.16.0
```

File: templates/.github/workflows/chatops.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ jobs:
           reaction-token: ${{ secrets.GITHUB_TOKEN }}
           repository: cloudposse/actions
           commands: rebuild-readme, terraform-fmt
-          permission: none
+          permission: triage
```
Comment (Nuru) [inline templates/.github/workflows/chatops.yml:18]: Maybe change slash-command-dispatch version above

**Review by Nuru — CHANGES_REQUESTED** [review]

File: .github/workflows/chatops.yml (hunk 1 above)
Comment (osterman) [reply .github/workflows/chatops.yml]: Ah, yes, but after we merge the upstream pr.

File: templates/.github/workflows/chatops.yml (hunk 3 above)
Comment (osterman) [reply templates/.github/workflows/chatops.yml:18]: Yup, deleted `slash-command-dispatch`

**Pull request merged**